webserver/src/main/java/io/kestra/webserver/controllers/WorkerInstanceController.java

```diff
@@ -0,0 +1,29 @@
+package io.kestra.webserver.controllers;
+
+import io.kestra.core.repositories.WorkerHeartbeatRepositoryInterface;
+import io.kestra.core.runners.WorkerHeartbeat;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.exceptions.HttpStatusException;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.inject.Inject;
+
+import java.util.List;
+
+@Controller("/api/v1/workers")
+@Requires(bean = WorkerHeartbeatRepositoryInterface.class)
+public class WorkerInstanceController {
+    @Inject
+    private WorkerHeartbeatRepositoryInterface workerHeartbeatRepositoryInterface;
+
+    @ExecuteOn(TaskExecutors.IO)
+    @Get(produces = MediaType.TEXT_JSON)
+    @Operation(tags = {"Workers"}, summary = "Get all workers")
+    public List<WorkerHeartbeat> findAll() throws HttpStatusException {
```

Comment:
The worker instance list will return worker heartbeats.
I'm not very fan of that as we didn't respect the naming.

As one extends the other we can makes the class generic and returns a list of instance with Elasticsearch and a list of Heartbeat in JDBC. This would be more elegant and avoid mapping between one and the other object.